Dockerfile-base

```diff
@@ -0,0 +1,53 @@
+FROM opencog/opencog-dev:cli
+MAINTAINER Enkusellasie Wendwosen <enku@singularitynet.io>
+
+#Run apt-get in NONINTERACTIVE mode
+ENV DEBIAN_FRONTEND noninteractive
+
+ENV HOME /root
+WORKDIR $HOME
+
+#Install pyenv and use it for managing python version
+RUN curl -L https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-installer | bash
+ENV PYENV_ROOT $HOME/.pyenv
+ENV PATH $PYENV_ROOT/shims:$PYENV_ROOT/bin:$PATH
+RUN pyenv install 3.5.2
+RUN pyenv virtualenv 3.5.2 general
+RUN pyenv global general
+ENV PYTHONPATH /usr/local/lib/python3.5/dist-packages:$PYTHONPATH
+
+
+RUN git clone https://github.com/enku-io/agi-bio.git
+WORKDIR agi-bio
+RUN mkdir build
+WORKDIR build
+RUN \
+    cmake .. && \
+    make && \
+    make install
+
+
+WORKDIR $HOME
+RUN git clone https://github.com/Habush/guile-json.git
+WORKDIR guile-json
+RUN \
+    apt-get install -y dh-autoreconf && \
+    autoreconf -vif && \
+    ./configure  && \
+    make && \
+    make install
+
+
+WORKDIR $HOME
+ENV SNET_DAEMON_V 0.1.6
+RUN mkdir snet-daemon-v$SNET_DAEMON_V
+RUN wget https://github.com/singnet/snet-daemon/releases/download/v$SNET_DAEMON_V/snet-daemon-v$SNET_DAEMON_V-linux-amd64.tar.gz
+RUN tar -xzf snet-daemon-v$SNET_DAEMON_V-linux-amd64.tar.gz -C snet-daemon-v$SNET_DAEMON_V --strip-components 1
+RUN ln snet-daemon-v$SNET_DAEMON_V/snetd snetd
+RUN rm snet-daemon-v$SNET_DAEMON_V-linux-amd64.tar.gz
+
+
+#setup grpc proxy
```

Comment:
We need to use the latest version of gRPC to add some options like write time outs. Check out the Dockerfile [here](https://github.com/MOZI-AI/moses-service/blob/312462fadcba1c6827208b8331b97066e8794eca/Dockerfile#L32) and change it to something similar